views/pages/profile.ejs

```diff
@@ -8,20 +8,15 @@
             <div class="subtitle">
                 <div class="card">
                     <div class="card-content">
```

Comment:
can you factorise the page structure with [ejs inheritance](https://stackoverflow.com/a/48046556/2590861)